commands/util.go

```diff
@@ -273,11 +273,15 @@ func makeDefaultHeader(collection interface{}) string {
 }
 
 func stripTimestamp(log string) (strippedLog string) {
-	regex := regexp.MustCompile("[a-zA-Z0-9\\s]*(stdout|stderr):\\s(.*)")
+	// parses out the timestamp if it exists first
+	// the timestamp expected format is YYYY-MM-DDTHH:MM:SS.[0-9]+Z
+	// an optional " stdout" or " stderr" stream identifier
+	// and the rest as the log line
+    regex := regexp.MustCompile("\\d{4}-[01]{1}\\d{1}-[0-3]{1}\\d{1}T[0-2]{1}\\d{1}:[0-6]{1}\\d{1}:[0-6]{1}\\d{1}.\\d+Z( (stdout|stderr):)?\\s(.*)")
```

Comment:
this regex does make a semantic change.
previously, the following line

`anything stdout: this is stdout` 
would be stripped to 
`this is stdout`

but with this patch the line will remain intact.